.github/workflows/main.yml

```
@@ -0,0 +1,24 @@
+name: ci
+
+on: [pull_request, push]
+
+jobs:
+  tests:
+    name: Tests
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [windows-latest, ubuntu-latest, macos-latest]
+        node: [14, 15]
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2-beta
+        with:
+          node-version: ${{matrix.node}}
+      - run: yarn install
+      - run: yarn build
+      # - name: Lint
+      #   run: yarn lint
+      - name: Unit tests
+        run: yarn test
```

Sir! This is an npm!
```suggestion
      - run: npm install
      - run: npm run build
      # - name: Lint
      #   run: npm run lint
      - name: Unit tests
        run: npm test
```

---

yeah, my bad! we have `package-lock`